src/index.js

```diff
@@ -129,10 +131,14 @@ export default function onClickOutsideHOC(WrappedComponent, config) {
         if (!events.forEach) {
           events = [events];
         }
+
+        const touchEvents = ['touchstart', 'touchmove'];
```

Comment:
as a constant, declaring it in this scope means it'll get redeclared despite never changing. Let's move this up to where `IGNORE_CLASS_NAME` gets declared (but without exporting it)